### PR TITLE
Update the “Remote-Ingestor” in the PSI Deployment branch

### PR DESCRIPTION
**Pull request by dwiessner-unibe**
## Description
Adjustments to the Remote-Ingestor according to OpenEM Alpha Release

## Summary by Sourcery

Implement the 'Ingestor' feature module to facilitate data ingestion, including metadata extraction and transfer management, and integrate it into the application's routing and navigation.

New Features:
- Introduce a new 'Ingestor' feature module for handling data ingestion processes, including metadata extraction and transfer management.

Enhancements:
- Add a new route and navigation link for the 'Ingestor' feature in the application header.

Build:
- Add new dependencies '@jsonforms/angular' and '@jsonforms/angular-material' to support JSON schema-based forms.

Deployment:
- Integrate the 'Ingestor' feature module into the application's routing system, allowing lazy loading of the module.

### CODE REVIEW

File: src/app/ingestor/ingestor/ingestor.component.ts [hunk 1]
```diff
@@ -88,19 +90,36 @@ export class IngestorComponent implements OnInit {
...
+    const params: any = {
+      page: page.toString(),
+      pageSize: pageSize.toString(),
     };
+    if (transferId) {
+      params.transferId = transferId;
+    }
+    this.http.get(this.connectedFacilityBackend + INGESTOR_API_ENDPOINTS_V1.TRANSFER, { params }).subscribe(
+      response => {
+        console.log('Transfer list received', response);
```
Comment (sbliven) [inline src/app/ingestor/ingestor/ingestor.component.ts:103]: Should be Logger.log (also elsewhere)

**Review by sbliven — APPROVED** [review]
I think this is fine for merging into psi-deployment. I've marked a few things I noticed with a quick scan that should be fixed before proposing to merge into `master`.

**Comment by sbliven** [comment]
@dwiessner-unibe Can you rebase and squash remote-ingestor onto master (b0b31e7), leaving out the release-jobs part? I think this PR should be against master, and then the psi-deployment branch is just a thin release branch incorporating the PRs we want to deploy right now at PSI.

**Comment by dwiessner-unibe** [comment]
Covered by https://github.com/SciCatProject/frontend/pull/1693

**Pull request closed without merging**